<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWizard</class>
 <widget class="QWizard" name="ExportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="modePage">
   <property name="title">
    <string>Export</string>
   </property>
   <property name="subTitle">
    <string>Select the target.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCommandLinkButton" name="clImage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="clWallpaper">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Wallpaper</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="clPosteRazor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Poster with PosteRazor</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="clPrint">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="clPdf">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="clSvg">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Scalable Vector Graphics</string>
      </property>
      <property name="nextPageId" stdset="0">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wallpaperPage">
   <property name="title">
    <string>Wallpaper</string>
   </property>
   <property name="subTitle">
    <string>Click the Finish button to change your wallpaper.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Scale Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="wbZoom">
         <property name="text">
          <string>Zoomed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wbScaleKeep">
         <property name="text">
          <string>Scaled, keep proportions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wbScaleIgnore">
         <property name="text">
          <string>Scaled</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wbOriginal">
         <property name="text">
          <string>Original Size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="imagePage">
   <property name="title">
    <string>Image</string>
   </property>
   <property name="subTitle">
    <string>Your work will be saved to a picture. Please fill the following options and click Finish.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Size and Orientation</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="saveWidth">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pixel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="saveHeight">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="imgFromCanvas">
           <property name="text">
            <string>Reset to Canvas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="imgFromDpi">
           <property name="text">
            <string>From Printing Dpi</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="saveLandscape">
         <property name="text">
          <string>Landscape</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Scale Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="ibZoom">
         <property name="text">
          <string>Zoomed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ibScaleKeep">
         <property name="text">
          <string>Scaled, keep proportions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ibScaleIgnore">
         <property name="text">
          <string>Scaled</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="imgAsIsBox">
      <property name="text">
       <string>as you see it (includes tools, lines, etc)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="posterazorPage">
   <property name="title">
    <string>PosteRazor</string>
   </property>
   <property name="subTitle">
    <string>Print Posters at home.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>The PosteRazor cuts a raster image into pieces which can afterwards be printed out and assembled to a poster.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="prWebLabel">
        <property name="text">
         <string>PosteRazor home (external website)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Click Finish to start PosteRazor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="prTutorialLabel">
        <property name="text">
         <string>How to use PosteRazor (external tutorial video)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="printPage">
   <property name="title">
    <string>Print Picture</string>
   </property>
   <property name="subTitle">
    <string>Select the paper options and click Finish to print.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">4</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Size and Orientation</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="printWidth">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QComboBox" name="printUnity">
         <item>
          <property name="text">
           <string>pixels</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>inches</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="printHeight">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="toolTip">
          <string>Dot Per Inch</string>
         </property>
         <property name="text">
          <string>Dpi:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="printDpi">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="printLandscape">
         <property name="text">
          <string>Landscape paper orientation</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="printKeepRatio">
         <property name="text">
          <string>Keep canvas proportion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="svgPage">
   <property name="title">
    <string>SVG Vector Image</string>
   </property>
   <property name="subTitle">
    <string>Export to the Scalable Vector Graphics image format.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">5</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QCheckBox" name="svgAsIsBox">
      <property name="text">
       <string>as you see it (includes tools, lines, etc)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="pdfPage">
   <property name="title">
    <string>PDF Export</string>
   </property>
   <property name="subTitle">
    <string>Export to PDF file.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">6</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QGroupBox" name="pdfBox">
      <property name="title">
       <string>Output Options</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Paper Size:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pdfPageButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="pdfScaleCombo">
         <item>
          <property name="text">
           <string>Original</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fit the page</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="pdfPosCombo">
         <item>
          <property name="text">
           <string>Top Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="pdfRes">
         <property name="suffix">
          <string> dpi</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>3200</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pdfPreviewButton">
      <property name="text">
       <string>Preview ...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
